### DISCUSSION TITLE
Integração com o Asaas

### DISCUSSION THREAD

**Discussion by SouVisto** [RamonSilva20/mapos#2013] [General]
Bom dia

Estou indo no Asaas - gero a Api Kay e vou no arquivo de pagamento e colo a Chave la no local indicado... Só que quando eu vou no sistema para gerar o Boleto ou Link ele vem com a frase ERRO AO CHAMAR O ASAAS.
Sou novo aqui alguém poderia me ajudar com esse problema.
Obs.: estou com a versão mais nova instalada.

**Comment by willph** [comment]
> Bom dia
> 
> Estou indo no Asaas - gero a Api Kay e vou no arquivo de pagamento e colo a Chave la no local indicado... Só que quando eu vou no sistema para gerar o Boleto ou Link ele vem com a frase ERRO AO CHAMAR O ASAAS. Sou novo aqui alguém poderia me ajudar com esse problema. Obs.: estou com a versão mais nova instalada.

Qual modo está usando? 

Sandbox ou produção?

**Reply by SouVisto** [reply]
Como sou novo no sistema como mudo ou vejo o modo que estou usando.
Desculpa minha ignorância na pergunta.

**Comment by willph** [comment]
No arquivo onde inseri a chave na linha 76        `'production' => false,` só alterar de `false` para `true`

**Reply by SouVisto** [reply]
Muito obrigado meu amigo, ajudou bastante e solucionou meu Problema e os demais quando precisarem.. Feliz ano Novo.

**Comment by Deniizynhu** [comment]
Tive que comentar aqui!
Procurei que nem um louco essa solução (sou iniciante em programação, que nem é a minha área mesmo) e você me fala que somente 4 letras resolve um problema de quase 10 horas de pesquisa?
Super fantástico, ajudou demais.
Forte abraço e muito sucesso na sua caminhada.
